README.md

```diff
@@ -6,66 +6,62 @@
 
+Suppose we sample a 1D harmonic oscillator from a few thermodynamic states:
 ```python
->>> from pymbar import MBAR, testsystems
->>> x_k, u_kn, N_k, s_n = testsystems.HarmonicOscillatorsTestCase().sample()
->>> mbar = MBAR(u_kn, N_k)
+>>> from pymbar import testsystems
+>>> [x_n, u_kn, N_k, s_n] = testsystems.HarmonicOscillatorsTestCase().sample()
 ```
+We have the `nsamples` sampled oscillator positions `x_n` (with samples from all states concatenated), [reduced potentials](http://www.alchemistry.org/wiki/Multistate_Bennett_Acceptance_Ratio#Reduced_potential) in the `(nstates,nsamples)` matrix `u_kn`, number of samples per state in the `nsamples` array `N_k`, and indices `s_n` denoting which thermodynamic state each sample was drawn from.
 
-Next, we extract the dimensionless free energy differences and uncertainties:
-
+To analyze this data, we first initialize the `MBAR` object:
+```python
+>>> mbar = MBAR(u_kn, N_k)
+```
+Estimating dimensionless free energy differences between the sampled thermodynamic states and their associated uncertainties (standard errors) simply requires a call to `getFreeEnergyDifferences()`:
 ```python
 >>> (Deltaf_ij_estimated, dDeltaf_ij_estimated, Theta_ij) = mbar.getFreeEnergyDifferences()
 ```
-
-or compute expectations of given observables A(x) for all states:
-
+Expectations and associated uncertainties can easily be estimated for observables `A(x)` for all states:
 ```python
->>> (A_k_estimated, dA_k_estimated) = mbar.computeExpectations(x_k)
+>>> A_kn = x_kn # use position of harmonic oscillator as observable
+>>> (EA_k, dEA_k) = mbar.computeExpectations(A_kn)
 ```
-See the help for these individual methods for more information on exact usage; in Python or IPython, you can view the docstrings with `help()`.  
-Additional examples can be found in [pymbar-examples](http://github.com/choderalab/pymbar-examples/).  Note that the example for free energy calculations found in pymbar-examples/alchemical-free-energy has moved to https://github.com/MobleyLab/alchemical-analysis
-
-
-Prerequisites
--------------
-
-To install and run pymbar requires the following:
-
-* Python 2.7 or later: http://www.python.org/
-* NumPy: http://numpy.scipy.org/
-* SciPy: http://www.scipy.org/
-* NumExpr: https://github.com/pydata/numexpr
-* six
-* nose
-* Some optional graphing functionality in the tests requires the matplotlib library: http://matplotlib.sourceforge.net/
+where `EA_k[k]` is the estimated expectation of the mean oscillator position in thermodynamic state `k`.
 
+See the docstring help for these individual methods for more information on exact usage; in Python or IPython, you can view the docstrings with `help()`.  
+Additional examples can be found in [pymbar-examples](http://github.com/choderalab/pymbar-examples/).  
```

Comment:
So... the `pymbar-examples` page lists the following at its header: 

> NOTE: THESE EXAMPLES HAVE BEEN MOVED TO THE MAIN PYMBAR REPOSITORY. FUTURE DEVELOPMENT WILL HAPPEN THERE.

and has not been updated since mid 2015, might not be a good idea to point people at this for now.